src/db/database.py

```
@@ -56,8 +60,9 @@ def check_connection():
 
 
 def create_all_tables():
-    """Create all tables defined in models."""
-    from src.db import models
+    """Create all tables defined in schema."""
+    # Import all models to ensure they're registered with Base
+    from src.db import schema  # noqa: F401 - import for side effects
```

Ruff (0.14.14)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)